string ingredient_id

int32 ingredient_type
---
# TODO: want any result?
---
# TODO: want any feedback?